api/signup_test.go

```diff
@@ -110,7 +110,7 @@ func (ts *SignupTestSuite) TestWebhookTriggered() {
 
 		u, ok := data["user"].(map[string]interface{})
 		require.True(ok)
-		assert.Len(u, 11)
+		assert.Len(u, 10)
```

Comment:
was this broken previously?

Reply:
Not sure if this assertion verifies anything important...

Reply:
yeah i think we can remove this check? it would be hard to figure out why this test is failing down the road if we don't set the right test config - i'm open to adding a comment as well if it makes things clearer

Reply:
Cool! Will remove it

Reply:
More generally, I'm not sure how much of the webhook_tests we use -- can revisit in the future.